x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/oidc/OpenIdConnectBaseRestHandler.java

```diff
@@ -33,7 +32,7 @@ protected Exception checkFeatureAvailable(RestRequest request) {
         Exception failedFeature = super.checkFeatureAvailable(request);
         if (failedFeature != null) {
             return failedFeature;
-        } else if (Realms.isRealmTypeAvailable(licenseState.allowedRealmType(), OIDC_REALM_TYPE)) {
+        } else if (licenseState.areStandardRealmsAllowed()) {
```

Comment:
I do not believe OIDC should be allowed in standard realms based on the `InternalRealms` class. It might be best to have these use Realms.isRealmTypeAvailable to try to keep the logic consolidated?

Reply:
This was an accidental change, I will revert.